src/openrct2-cli/Cli.cpp

```diff
@@ -27,7 +27,7 @@ using namespace OpenRCT2;
 int main(int argc, char * * argv)
```

Comment:
Can't the `argv` here be marked as `const`?

Reply:
@janisozaur I don't think you can change the signature of main.

Reply:
I'm pretty sure you can, see https://github.com/OpenRCT2/OpenRCT2/pull/6493

Reply:
Ok, if you say so, it just seems unusual to change the signature that is widely used. I would personally keep the cast.

Reply:
It is widely used, but it is not specified by the standard. The `argv` data is known to be coming from OS and should not really be touched, in fact, we had an issue due to that way in the past. We have yet to see if all the CIs come green on that other PR.

Reply:
Should I close this then?

Reply:
I'm happy for you to take credit, you can update this PR to match changes done in #6493 or I can mention you in the commit message. Up to you.

Reply:
Ok, I will update here.